gensim/models/keyedvectors.py

```diff
@@ -2512,3 +2185,23 @@ def _try_upgrade(wv):
             "from scratch."
         )
         wv.compatible_hash = False
+
+
+
+class Vocab(object):
+    """A single vocabulary item, used internally for collecting per-word frequency/sampling info,
+    and for constructing binary trees (incl. both word leaves and inner nodes).
+
+    """
+    def __init__(self, **kwargs):
+        self.count = 0
+        self.__dict__.update(kwargs)
+
+    def __lt__(self, other):  # used for sorting in a priority queue
+        return self.count < other.count
+
+    def __str__(self):
+        vals = ['%s:%r' % (key, self.__dict__[key]) for key in sorted(self.__dict__) if not key.startswith('_')]
+        return "%s(%s)" % (self.__class__.__name__, ', '.join(vals))
```

Comment:
```suggestion
        return "%s<%s>" % (self.__class__.__name__, ', '.join(vals))
```

By convention, we now use `<>` to delimit instance parameters in `str()`.
I know this PR carried this code over without change, just cleaning up.